### PR TITLE
Switch 404 default as per #43

### PR DESCRIPTION
**Pull request by cliftonc**
As discussed in #43 w/ @tomruttle

### CODE REVIEW

**Comment by tomruttle** [comment]
LGTM. Thanks for this. :+1:

File: src/middleware/htmlparser.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ function getMiddleware(config, reliableGet, eventHandler) {
                 cacheTTL = utils.timeToMillis(getCxAttr(fragment, 'cx-cache-ttl') || '1m'),
                 explicitNoCacheAttr = getCxAttr(fragment, 'cx-no-cache'),
                 explicitNoCache = req.explicitNoCache || (explicitNoCacheAttr ? eval(explicitNoCacheAttr) : false),
-                ignore404 = getCxAttr(fragment, 'cx-ignore-404') === 'true',
+                ignore404 = (getCxAttr(fragment, 'cx-ignore-404') || 'true')  === 'true',
```
Comment (tomruttle) [inline src/middleware/htmlparser.js]: Ha!

**Comment by dmorgantini** [comment]
@cliftonc / @tomruttle should we merge or close this PR?

**Comment by tomruttle** [comment]
I vote merge. But with a semver bump.

**Pull request merged**